src/main.go

```diff
@@ -31,21 +55,15 @@ func main() {
 		log.Fatal(err)
 	}
 
-	fmt.Println("You win if you :")
-	for _, result := range(results.Win) {
-		fmt.Printf(
-			"Place a marble in %d %d and rotate quadrant %v in %v \n",
-			result.PlaceMarble[0],
-			result.PlaceMarble[1],
-			result.Rotate[0],
-			result.Rotate[1],
-		)
-	}
+	sort.Slice(
+		results,
+		func(i, j int) bool { return results[i].Score > results[j].Score },
+	)
 
-	fmt.Println("\nYou loose if you :")
-	for _, result := range(results.Loose) {
+	for _, result := range(results[:MAX_RESULTS]) {
```

Comment:
Why do you limit results to 10?

Reply:
It was asked in the card 

> The solver should give me the 5 best moves to play

 but I increased it to 10 because first results are often close e.g. the picture in the PR

Reply:
Maybe you should try to improve the scoring method rather than limiting to 10.

Reply:
I don't think it's a scoring problem but due to the game itself.

For example in this case :

```
─────┐
|0|1|0||0|0|0|
|0|1|0||0|0|0|
|0|1|0||0|0|0|
|─────|
|0|1|0||0|0|0|
|0|0|0||0|0|0|
|0|0|0||0|0|0|
─────┘
```

You have just to put one marble in 4, 1 and rotate quarter 1 or quarter 3 (because it doesn't change the alignment)
The CLI will return me that I got 4 ways to win:
- Put marble in 4,1 and rotate 1 clockwise
- Put marble in 4,1 and rotate 1 counter clockwise
- Put marble in 4,1 and rotate 3 clockwise
- Put marble in 4,1 and rotate 3 counter clockwise

So only 5 results is not completely relevants at this step.
By the way I have to cut some part of my detection tree (w/ Alpha Beta pruning). It should return less duplicated results.

Reply:
Can't wait to see the Alpha Beta pruning in action :P